types/hapi/test/route/additional-options.ts

```diff
@@ -37,3 +60,9 @@ var cache: Hapi.RouteCacheOptions = {
     expiresAt: '22:44',
 };
 */
+
+var payloadOptions: Hapi.RoutePayloadConfigurationObject = {
+  allow: 'multipart/form-data',
+  maxBytes: 123,
+  output: 'file',
+};
```

Comment:
We could do the same by adding:  `payload: {},` to the top declaration:

```ts
var authConfig: Hapi.RouteAdditionalConfigurationOptions = {
    app: {},
    payload: {},
};
```

And good catch, thanks.